src/app/pages/fund/fund.layout.tsx

```diff
@@ -40,9 +40,7 @@ export function FundLayout(props: FundLayoutProps) {
           textAlign={['left', 'center']}
         >
           {/* TODO: Investigate removing / refactoring <PageTitle */}
-          <styled.h1 textStyle="heading.02">
-            {desktopViewport ? 'Let’s get STX into your wallet' : 'Get STX'}
-          </styled.h1>
+          <styled.h1 textStyle="heading.02">Let's get STX into your wallet</styled.h1>
```

Comment:
We want to remove the reference to STX altogether. @mica000 @markmhx

Reply:
Updated